cockroachcloud/cloud-sso.md

```diff
@@ -25,7 +25,7 @@ With support for SAML and OIDC, {{ site.data.products.db }} allows enterprise or
 
 ### Custom sign-in page
 
-With this feature, customers get a unique and private sign-in URL for their {{ site.data.products.db }} organization, and will no longer use the [public sign-in URL](https://cockroachlabs.cloud).
+With Enterprise Authentication, customers get a unique and private sign-in URL for their {{ site.data.products.db }} organization, and will no longer use the [public sign-in URL](https://cockroachlabs.cloud).
```

Comment:
```suggestion
When {{ site.data.products.enterprise }} Authentication is enabled, members of your {{ site.data.products.db }} 
 organization use a unique and private sign-in page and can no longer sign in using the [public sign-in URL](https://cockroachlabs.cloud).
```
A little rewording to make this a bit more direct, reflect the same language as the section title, and avoid future tense (and fix the linter problem)